src/components/comments/comments.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import { getCommentsByArticleId } from '../../api/api'
+import CommentCard from '../comments/CommentCard'
+
+
+const Comments = () => {
+    const { article_id } = useParams();
+    const [comments, setComments] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+
+    useEffect(() => {
+        const fetchComments = async () => {
+          try {
+            setIsLoading(true);
+            const commentData = await getCommentsByArticleId(article_id);
+            setComments(commentData);
+          } catch(err) {
+            console.error('Error fetching comments', err)
```

Comment:
Good to see that you are thinking about errors - use these to display some meaningful feedback to the user.